axis_backend/apps/clients/serializers/__init__.py

```
@@ -0,0 +1,24 @@
+)
+
+__all__ = [
+    'IndustryListSerializer',
+    'IndustryDetailSerializer',
+    'IndustryCreateSerializer',
+    'IndustryUpdateSerializer',
+    'ClientListSerializer',
+    'ClientDetailSerializer',
+    'ClientCreateSerializer',
+    'ClientUpdateSerializer',
+]
```

Ruff (0.14.5)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)